frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -247,6 +247,65 @@ const AllVariableDropdowns = () => {
   );
 };
 
+const DashboardDropdown = ({ items, onChange, selectedKey }) => {
+  const { t } = useTranslation();
+
+  const [isOpen, toggleIsOpen, , setClosed] = useBoolean(false);
+
+  const tagColors: ('red' | 'purple' | 'blue' | 'green' | 'cyan' | 'orange')[] = [
+    'red',
+    'purple',
+    'blue',
+    'green',
+    'cyan',
+    'orange',
+  ];
+
+  const allTags = _.flatMap(items, 'tags');
+  const uniqueTags = _.uniq(allTags);
```

Comment:
Is the order of tags in the array going to be the same every time? It might be odd if the tag colors are different each time you visit the page.

Reply:
Yes, it should be preserved because `_.uniq` preserves the order (see https://lodash.com/docs/4.17.15#uniq).